ben02/inicon_ben02.F

```diff
@@ -65,8 +71,21 @@ subroutine icaux_ben02
           ticem(i,j)=tsrfm(i,j)
           tml_tda(i,j)=temp(i,j,1)+t0deg
           sml_tda(i,j)=saln(i,j,1)
-          alb_tda(i,j)=albs_f*ficem(i,j)+albw_d*(1.-ficem(i,j))
```

Comment:
change of variable: "albw_d" -> "albdif" (imported from mod_ben02)

Reply:
The variables you mention are declared in "phy/common_forc.h" of the current master (some are renamed) but are specific to the experiment configurations "ben02syn" and "ben02clim". Therefore I decided to declare them in modules contained in the "ben02" folder. They are not used when BLOM is coupled to NorESM. I have tested that the "ben02syn" and "ben02clim" configurations also produce bit-identical results.